source/Applications/Advanced/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp

```diff
@@ -97,10 +127,45 @@ int main()
         std::cout << "Computing camera pose in robot base frame" << std::endl;
         const Eigen::MatrixXf transformBaseToCamera = transformBaseToEndEffector * transformEndEffectorToCamera;
 
-        std::cout << "Computing (picking) point in robot base frame" << std::endl;
-        const Eigen::Vector4f pointInBaseFrame = transformBaseToCamera * pointInCameraFrame;
-        std::cout << "Point coordinates in robot base frame: " << pointInBaseFrame.x() << " " << pointInBaseFrame.y()
-                  << " " << pointInBaseFrame.z() << std::endl;
+        switch(enterCommand())
+        {
+            case CommandType::cmdTransformSinglePoint:
+            {
+                // define (picking) point in camera frame
+                const Eigen::Vector4f pointInCameraFrame(81.2F, 18.0F, 594.6F, 1.0F);
+                std::cout << "Point coordinates in camera frame: " << pointInCameraFrame.x() << " "
+                          << pointInCameraFrame.y() << " " << pointInCameraFrame.z() << std::endl;
+
+                std::cout << "Computing (picking) point in robot base frame" << std::endl;
+                const Eigen::Vector4f pointInBaseFrame = transformBaseToCamera * pointInCameraFrame;
+                std::cout << "Point coordinates in robot base frame: " << pointInBaseFrame.x() << " "
+                          << pointInBaseFrame.y() << " " << pointInBaseFrame.z() << std::endl;
+                break;
+            }
+            case CommandType::cmdTransformPointCloud:
+            {
+                const auto dataFile = std::string(ZIVID_SAMPLE_DATA_DIR) + "/ZividGem.zdf";
+                std::cout << "Reading ZDF frame from file: " << dataFile << std::endl;
+                const Zivid::Frame frame = Zivid::Frame(dataFile);
+
+                std::cout << "Getting point cloud from frame" << std::endl;
+                auto pointCloud = frame.pointCloud();
+
+                std::cout << "Converting to Zivid matrixes to transform entire point cloud" << std::endl;
```

Comment:
typo -> matrixes - should be matrix
Suggestion: write just "Transforming the point cloud" instead of  "Converting to Zivid matrixes to transform entire point cloud"

Reply:
done. Changed the output to "Transforming the point cloud"